monitoring/mock_uss/scdsc/routes_injection.py

```diff
@@ -251,24 +273,29 @@ def delete_flight(flight_id: str) -> Tuple[str, int]:
             ),
             200,
         )
-    scd_client.notify_subscribers(
-        resources.utm_client,
-        result.operational_intent_reference.id,
-        None,
-        result.subscribers,
-    )
-
+    for subscriber in result.subscribers.copy():
+        if subscriber.uss_base_url == 'http://host.docker.internal:8074/mock/scd' :
```

Comment:
Why would we not want to notify these subscribers?

Reply:
This is the mock_uss itself. We were getting POSTs to self in mock_uss. Whereas, we were looking for POSTs from the other USS (USS-under-test) to mock_uss.

Reply:
Perhaps self-interactions are not interesting for message signing, but mock_uss is used for many other purposes apart from message signing.  The general expectation is that notifications will be sent to all subscribers and the message signing component is free to ignore incoming notifications from itself if that is the appropriate thing to do with regard to message signing.  But, message signing should not change the baseline behavior of mock_uss.